packages/catalog-realm/catalog-app/listing/listing.gts

```diff
@@ -45,6 +48,7 @@ class EmbeddedTemplate extends Component<typeof Listing> {
   @tracked selectedAccordionItem: string | undefined;
   @tracked writableRealms: { name: string; url: string; iconURL?: string }[] =
     [];
+  @tracked roomId: string | null = null;
```

Comment:
how about we just use matrix service `getRoomId`. so we know we are getting state from one place. If matrix service has another way to know about the roomId, we don't dig into internals of the window storage. 

You can either add a new command that wraps AddSkillsToRoom or have a field that suggests to look for roomId inside current room

Reply:
Or use UseAiAssisstantCommand and add the current room feature there